### ISSUE TITLE
In debian bookwork kiosk autostart doesn't work

### ISSUE THREAD

**Issue by douglashetherington** [Snapp-Embedded/snapp_installer#5]
Love this project - great work thank you!

It looks like the way autostart works has changed in bookworm - See this thread: https://forums.raspberrypi.com/viewtopic.php?t=357858

`$HOME/.config/lxsession/LXDE-pi/autostart` now seems to be ignored.

It looks like it's now using `~/.config/wayfire.ini` to look for autostart items adding:
```
[autostart]
kiosk=bash /home/douglash/kiosk/kiosk.sh && sleep 5 && xdotool key alt+F11
```
to the bottom of that file launches the app.

**Comment by douglashetherington** [comment]
Hi @payam-zahedi,

It successfully launches the app but full screen didn't work. I just quickly ended up changing the `my_application.cc` file in the flutter project to add

```
  gtk_window_set_default_size(window, 1280, 720);
  gtk_window_fullscreen(GTK_WINDOW(window)); // <--- Added this line
  gtk_widget_show(GTK_WIDGET(window));
```

Which makes it work. I also added the declutter command to the run (which is commented out in the autostart script) which removes the cursor as well.

I came across [Cage](https://github.com/cage-kiosk/cage/wiki/) - i've not tried it yet but it could be what you guys want.

**Comment by payam-zahedi** [comment]
Thanks @douglashetherington 

we will look into this issue and will let you know about the future changes

**Comment by payam-zahedi** [comment]
In the newest version of the Raspberry Pi OS, the default display manager is Wayland. So all the possibilities of X11 will not work.

I fixed this issue in #6 PR, so we are support both Wayland and X11.

As You may have noticed, there is only a problem with full screen mode.

Under Wayland, certain keyboard features like alt + F11 won't work to make apps full screen. Checkout [this issue](https://github.com/flutter/flutter/issues/138338)

I added another command for the full screen issue. You can run the `enable_fullscreen` in the root of your flutter project to change your setting in your app.

Let me know if you need more information.
